CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ endif()
 option(CUDA    "Enable support for CUDA")
 option(UMESIMD "Enable UME::SIMD backend")
 option(VC      "Enable Vc backend")
+option(VECRNG  "Enable support for VecRng")
```

Comment:
I am not sure what is the intention with VecRng, if it will be temporarily in VecCore or permanently. I would say that if it will be moved in for good, there should not be the option of disabling it, but it should always be installed.

Reply:
At we agreed, VecRng resides under vecrng temporarily.  So the purpose of the option
is exactly not to disturb anything to vecrng.